embed-code-go/test/embedding_instruction_test/embedding_instruction_test.go

```diff
@@ -183,21 +174,18 @@ func TestFragmentAndStart(t *testing.T) {
 
 	xmlString := buildInstruction("org/example/Hello.java", instructionParams)
 	config := buildConfigWithPreparedFragments()
-	embedding_instruction.FromXML(xmlString, config)
 
-	preparator.cleanup()
+	_, err := embedding_instruction.FromXML(xmlString, config)
+	if err == nil {
+		t.Errorf("FromXML had to raise a panic, but it didn't.")
```

Comment:
Please review period usage in your `Errorf` some of them have this period while others not.